conf/logs.json

```diff
@@ -308,5 +308,524 @@
         }
       }
     }
-  }
-}
\ No newline at end of file
+  },
+  "bro:conn.log": {
+    "parser": "json",
+    "schema": {
+      "ts": "string",
+      "uid": "string",
+      "id": "string",
+      "proto": "string",
+      "service": "string",
+      "duration": "string",
+      "orig_bytes": "integer",
+      "resp_bytes": "integer",
+      "conn_state": "string",
+      "local_orig": "boolean",
```

Comment:
The Bro documentation shows this as a `bool` type (per https://www.bro.org/sphinx/scripts/base/protocols/conn/main.bro.html#type-Conn::Info). It says:

```
If the connection is originated locally, this value will be T. If it was originated remotely it will be F. In the case that the Site::local_nets variable is undefined, this field will be left empty at all times.
```

As you can see, the possible values are `T` and `F`, which aren't real boolean values. This is confirmed here too: https://www.bro.org/sphinx/script-reference/types.html#type-bool

As a result, these need to be of type `string`. Please check what other fields might be impacted and how Bro's concept of types differs: https://www.bro.org/sphinx/script-reference/types.html#type-bool